assets/css/amp-editor-story-blocks.css

```diff
@@ -37,6 +37,42 @@
 	display: none !important;
 }
 
+/* Quote block adjustments */
+.wp-block-quote__citation {
+	font-size: 16px;
+	margin-top: 24px;
+	display: block;
+}
+
+.editor-styles-wrapper .wp-block-quote p {
+	line-height: 34px;
+	font-weight: bold;
+	font-size: 30px;
+	font-family: "Source Sans Pro", sans-serif;
+}
+
+.wp-block-quote,
+.wp-block-quote[style*="text-align: right"] {
+	border: none;
+	padding-top: 100px;
+}
+
+.block-editor-block-list__layout .block-editor-block-list__block [data-type="core/quote"] .block-editor-block-list__block-edit::before {
+	right: -5px;
+	left: -5px;
+	top: -5px;
+	bottom: -5px;
+}
+
+.editor-block-styles__item.block-editor-block-styles__item[aria-label="White"] .block-editor-block-styles__item-preview,
```

Comment:
That doesn't work on localized sites. Is there any other way to target the element here?

Reply:
Actually, we can remove this one rule, since the inner block preview is already targeted. That made it look a bit better but it's true that in some cases it wouldn't work.

Without it, it looks like this:
![Screen Shot 2019-04-10 at 3 58 22 PM](https://user-images.githubusercontent.com/3294597/55885052-81fdb800-5ba9-11e9-8b2a-6af667e21b74.png)